add 0.1.6 release note (#244)
streamnative/function-mesh: CHANGELOG.md

# Function Mesh v0.1.6 Release Notes

## v0.1.6 What's New

- Move function runner image to java11 to support Pulsar 2.8 ([#202](https://github.com/streamnative/function-mesh/pull/202))
- Fixed max replicas value in mesh worker service ([#212](https://github.com/streamnative/function-mesh/pull/212))
- Update sink/source component observation logic ([#217](https://github.com/streamnative/function-mesh/pull/217))

> - Add function/sink/source status API support with mesh worker service ([#220](https://github.com/streamnative/function-mesh/pull/220)) ([#224](https://github.com/streamnative/function-mesh/pull/224)) ([#225](https://github.com/streamnative/function-mesh/pull/225))

Reviewer: ```suggestion
- Support Pulsar Functions and connectors (sinks and sources) status API in Function Mesh Worker service. ([#220](https://github.com/streamnative/function-mesh/pull/220)) ([#224](https://github.com/streamnative/function-mesh/pull/224)) ([#225](https://github.com/streamnative/function-mesh/pull/225))
```

> - Allow set service account name for running pod ([#226](https://github.com/streamnative/function-mesh/pull/226))

Reviewer: ```suggestion
- Support setting the service account name for running Pods. ([#226](https://github.com/streamnative/function-mesh/pull/226))
```

> - Fixed NPE when not set AutoAck ([#234](https://github.com/streamnative/function-mesh/pull/234))

Reviewer: ```suggestion
- Fix the NPE that is triggered when the `AutoAck` option is not set. ([#234](https://github.com/streamnative/function-mesh/pull/234))
```

> - Fixed mesh worker service get status with fault pod ([#239](https://github.com/streamnative/function-mesh/pull/239))

Reviewer: ```suggestion
- Fix the issue that the Function Mesh worker service gets status with a fault pod. ([#239](https://github.com/streamnative/function-mesh/pull/239))
```

> - Remove logTopic from source/sink CRD ([#242](https://github.com/streamnative/function-mesh/pull/242))

Reviewer: ```suggestion
- Remove the `logTopic` option from source  and sink CRDs. ([#242](https://github.com/streamnative/function-mesh/pull/242))
```

> - Fixed service name observe ([#230](https://github.com/streamnative/function-mesh/pull/230))

Reviewer: ```suggestion
- Fix the observation logic of the service name. ([#230](https://github.com/streamnative/function-mesh/pull/230))
```

> - Bump Pulsar to 2.8 ([#240](https://github.com/streamnative/function-mesh/pull/240))

Reviewer: ```suggestion
- Bump Pulsar to version 2.8.0. ([#240](https://github.com/streamnative/function-mesh/pull/240))
```
